.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - '**'
+
+env:
+  SECP256K1_BENCH_ITERS: 2
+  SECP256K1_TEST_ITERS: 16
```

Comment:
Any reason not to keep the default of 64 for `TEST_ITERS`? We typically set lower values only in special environments such as QEMU.

Reply:
I cannot recall what exactly caused my decision. Going to address your comment and make a new push after addressing https://github.com/bitcoin-core/secp256k1/pull/1389#issuecomment-1671120227.

Reply:
Fixed.